source/base/DateTime.ooc

```diff
@@ -41,6 +41,7 @@ DateTime: cover {
 	_ticks: ULong = 0
 	ticks ::= this _ticks
 	init: func@ (=_ticks)
+	init: func@ ~fromNanoseconds (nanoseconds: ULong) { this _ticks = nanoseconds / This nanosecondsPerTick }
```

Comment:
This might be a bit unclear when you use it:
```ooc
time := DateTime new(1000)
```
Isn't it better to have a static function for it?
```ooc
time := DateTime fromNanoseconds(1000)
```